zulip_bots/zulip_bots/bots/merels/test/test_database.py

```diff
@@ -3,64 +3,68 @@
 from libraries import database
 from libraries import game_data
 from zulip_bots.simple_lib import SimpleStorage
-
-
-class DatabaseTest(unittest.TestCase):
+from zulip_bots.test_lib import BotTestCase, DefaultTests
+from zulip_bots.game_handler import GameInstance
+from merels import MerelsHandler
+
+#Original: class DatabaseTest(unittest.TestCase):
+class DatabaseTest(BotTestCase, DefaultTests):
+    bot_name = 'merels'
+    
     def setUp(self):
         self.storage = SimpleStorage()
-        self.merels = database.MerelsStorage(self.storage)
+        self.merels = database.MerelsStorage("", self.storage)
 
+    '''
+    REMOVE B/C No Games are "Created" anymore
     def test_create_duplicate_game(self):
         self.merels.create_new_game("test")
 
         self.assertEqual(self.merels.create_new_game("test"), False)
-
+    '''
+    
+    
```

Comment:
`pep8      | zulip_bots/zulip_bots/bots/merels/test/test_database.py:27:5: E303 too many blank lines (2)`
There are two blank lines but only one is desired.